### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/FieldTrainingLab#3] [General]
@JFYoung01 
thank you for the simplified Chinese translation!
![image](https://user-images.githubusercontent.com/39887717/134782397-e70ccde3-ad5f-4ac5-8aa7-fe0606b537ba.png) (中文) <zh-cn.cfg>

**Comment by JFYoung01** [comment]
**Simplified Chinese localization Update**

I recently took some time out and happened to notice that FTL had been updated, so I updated the Simplified Chinese localization for that purpose.
[zh-cn.txt](https://github.com/zer0Kerbal/FieldTrainingLab/files/12705839/zh-cn.txt)
